src/dashboard/style.scss

```diff
@@ -1,5 +1,9 @@
 @import "../blocks/plugins/feedback/editor.scss";
 
+:root {
+	--main-link-color: #2271b1;
```

Comment:
You can move this under `#otter` instead of `:root`

`:root` is useful when you have multiple _independent_ components that share the values inside the same page. Here is not the case.

Reply:
I moved the declaration, as suggested.